lib/matplotlib/axes/_axes.py

```diff
@@ -6812,8 +6812,14 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
              xmax], [ymin, ymax]]``. All values outside of this range will be
              considered outliers and not tallied in the histogram.
 
-        normed : bool, optional, default: False
-             Normalize histogram.
+        density : boolean, optional
+             If False, the default, plots and returns the number of samples
+             in each bin. If True, returns the probability *density*
+             function at the bin, ``bin_count / sample_count / bin_area``.
+             Default is ``None`` for both *normed* and *density*. If either is
```

Comment:
Can we keep the discussion of deprecated `normed` kwarg in the description of that kwarg?